index.d.ts

```diff
@@ -57,13 +58,33 @@ declare module "oncoprintjs"
         // either `colormap_name` or `colors` needs to be present
         colors?: string[]|number[][]; // hex, rgb, rgba | [r,g,b,a][]
         colormap_name?: string; // name of a colormap found in src/js/heatmapcolors.js
+        value_stop_points: number[];
```

Comment:
thanks for adding this. could you also please add that to the comment in oncoprintruleset.js line 615

Reply:
Added.